thunder/tests/test_inplace_copy.py

```diff
@@ -121,3 +122,40 @@ def forward(self, x):
     assert_close(net.state_dict()["dense1_bn.running_mean"], torch_net.state_dict()["dense1_bn.running_mean"])
     assert_close(net.state_dict()["dense1_bn.running_var"], torch_net.state_dict()["dense1_bn.running_var"])
     assert_close(x.grad, x1.grad)
+
+
+@instantiate(dtypes=(thunder.float32,))
+def test_inplace_copy_sanity_check(executor, device, dtype):
```

Comment:
@jjsjann123, could you please take a look at these test cases and tell us if these are sufficient restrictions to generate correctly functioning nvFuser code?

Reply:
tests looks fine~ thanks for the ping.